config/locales/en.yml

```diff
@@ -0,0 +1,47 @@
+---
+en:
+  date:
+    abbr_day_names:
+    - Sun
+    - Mon
+    - Tue
+    - Wed
+    - Thu
+    - Fri
+    - Sat
+    abbr_month_names:
+    - 
```

Comment:
Are empty lines needed..?

Reply:
thanks for the review @jhigman 🙏  
the missing month key value is in case a zero is passed (no 0th month) see
https://github.com/rails/rails/blob/main/activesupport/lib/active_support/locale/en.yml
I'll copy the comment over to the yml file.

Reply:
fyi - `i18n-tasks normalize` removes the comment